docs/cloud-storage-fileset-example.md

```diff
@@ -0,0 +1,678 @@
+
+- `gravitino-${aws,gcp,aliyun,azure}-bundle-{gravitino-version}.jar` are the jars that contain all the necessary dependencies to access the corresponding cloud storages. For instance, `gravitino-aws-bundle-${gravitino-version}.jar` contains the all necessary classes including `hadoop-common`(hadoop-3.3.1) and `hadoop-aws` to access the S3 storage.
+They are used in the scenario where there is no hadoop environment in the runtime.
+
+- If there is already hadoop environment in the runtime, you can use the `gravitino-${aws,gcp,aliyun,azure}-${gravitino-version}.jar` that does not contain the cloud storage classes (like hadoop-aws) and hadoop environment. Alternatively, you can manually add the necessary jars to the classpath.
+
+The following table demonstrates which jars are necessary for different cloud storage filesets:
+
+| Hadoop runtime version | S3                                                                                                                                       | GCS                                                                                                          | OSS                                                                                                                                        | ABS                                                                                                                |
+|------------------------|------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| No Hadoop environment  | `gravitino-aws-bundle-${gravitino-version}.jar`                                                                                          | `gravitino-gcp-bundle-${gravitino-version}.jar`                                                              | `gravitino-aliyun-bundle-${gravitino-version}.jar`                                                                                         | `gravitino-azure-bundle-${gravitino-version}.jar`                                                                  |
+| 2.x, 3.x               | `gravitino-aws-${gravitino-version}.jar`, `hadoop-aws-${hadoop-version}.jar`, `aws-sdk-java-${version}` and other necessary dependencies | `gravitino-gcp-{gravitino-version}.jar`, `gcs-connector-${hadoop-version}`.jar, other necessary dependencies | `gravitino-aliyun-{gravitino-version}.jar`, hadoop-aliyun-{hadoop-version}.jar, aliyun-sdk-java-{version} and other necessary dependencies | `gravitino-azure-${gravitino-version}.jar`, `hadoop-azure-${hadoop-version}.jar`, and other necessary dependencies |
```

Comment:
We don't have to use tables here. Markdown sucks when used to create tables, especially when the cell contents are long.
We can use, for example, unordered lists for this.